src/runtime/cuda/cuda_module.cc

```diff
@@ -173,10 +173,12 @@ class CUDAWrappedFunc {
  public:
   // initialize the CUDA function.
   void Init(CUDAModuleNode* m, ObjectPtr<Object> sptr, const std::string& func_name,
-            size_t num_void_args, const std::vector<std::string>& launch_param_tags) {
+            size_t num_void_args, const std::vector<std::string>& launch_param_tags,
+            bool has_programmatic_dependent_launch) {
```

Comment:
thanks a lot for the contribution, to align with existing impl, we can use launch param tag, see LaunchParamConfig implementation. The launch param tag can be specified as part of `tir.kernel_launch_params`, which then set the LaunchPramConfig setting

we can use 

constexpr const char* kUseProgramaticDependentLaunch = "tir.use_programtic_dependent_launch";